pkg/types/baremetal/validation/libvirt.go

```diff
@@ -0,0 +1,69 @@
+// +build baremetal
+
+package validation
+
+import (
+	"fmt"
+	"github.com/libvirt/libvirt-go"
+	"github.com/openshift/installer/pkg/types/baremetal"
+	"github.com/pkg/errors"
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	"strings"
+)
+
+func init() {
+	baremetalValidators = append(baremetalValidators, validateInterfaces)
+}
+
+var interfaces []libvirt.Interface
```

Comment:
let's not use global variables.

Reply:
I find package-scope cache variables easier to understand than a closure for a simple case like this, but I've removed it now, PTAL